src/Uno.UI/Uno.UI.Skia.csproj

```diff
@@ -111,4 +111,28 @@
 			<UndefineProperties>TargetFramework</UndefineProperties>
 		</ProjectReference>
 	</ItemGroup>
+
+	<Target Name="VS16Mac_RemoveSystemMemory" BeforeTargets="ResolveAssemblyReferences" Condition="'$(MSBuildVersion)' &gt;= '16.0' and $([MSBuild]::IsOsPlatform('OSX'))">
```

Comment:
This should not be required for .NET 5/6, only for xamarin. Which issues did you get ?

Reply:
When the Build on C.I. it report that the ArrayPull is present both in the corelibs and in the System.Buffers.